pojo.go

```diff
@@ -210,6 +201,50 @@ func RegisterPOJO(o POJO) int {
 	return structInfo.index
 }
 
+// easy for test
+func UnRegisterPOJOs(os ...POJO) []int {
+	arr := make([]int, len(os))
+	for i := range os {
+		arr[i] = UnRegisterPOJO(os[i])
+	}
+
+	return arr
+}
+
+func UnRegisterPOJO(o POJO) int {
+	pojoRegistry.Lock()
+	defer pojoRegistry.Unlock()
+
+	var structInfo structInfo
```

Comment:
you don't need to define a structInfo: 

`goName := obtainValueType(o).String()`

Reply:
done.